.markdownlint.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
The file must be in a central location, otherwise we have to repeat it in each repository.

Reply:
@froschdesign I have a basic one in the CI container; we only have to add one directly in repos when we want to modify that ruleset (generally to exclude specific rules for specfic files).